src/app/pages/voices/voices.component.ts

```
@@ -59,56 +59,52 @@ export class VoicesComponent implements OnInit {
   constructor(private utils:UtilsService, private sg:firebaseService) { }
   async ngOnInit() {
-    try {
-      this.getStoryOfWeek()
-      this.browserId = this.utils.getBrowserId();
-
-      const storyIds = this.story.map(s => s.id);
-
-      if (!storyIds.length) {
+    d3.json(`${environment.storageURL}/${environment.bucketName}/${environment.folderName}/${STORY_OF_THE_WEEK}`).then((data: any) => {
+      this.story= data["data"][0];
+      debugger;
```

Biome (2.1.2)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)